README.md

```diff
@@ -19,13 +19,138 @@ By default, the project will source the Synthea and OMOP vocabulary data from se
 
 Users are welcomed, however, to utilize their own Synthea and/or OMOP vocabulary tables as sources.  Instructions for the "BYO data" setup are provided below.
 
+We reccomend using uv to manage dependencies, particularly for development. Installation instructions can be found [here](https://docs.astral.sh/uv/getting-started/installation/)
+
 ### Prerequisites
 - See the top of [this page](https://docs.getdbt.com/docs/core/pip-install) for OS & Python requirements.  (Do NOT install dbt yet - see below for project installation and setup.)
 - It is recommended to use [VS Code](https://code.visualstudio.com/) as your IDE for developing this project.  Install the `dbt Power User` extension in VS Code to enjoy a plethora of useful features that make dbt development easier
 - This project currently only supports **Synthea v3.0.0**
 - Python **3.12.0** is the suggested version of Python for this project.
 
-### Repo Setup
+### Setup -uv (recommended)
+ 1. Clone this repository to your machine
+ 2. `cd` into the repo directory and set up a virtual environment:
+
+ If using duckdb:
+ ```bash
+ uv sync --group duckdb
+ ```
+
+ If using postgres:
+ ```bash
+ uv sync --group postgres
+ ```
+
+ If you want to add the dev dependencies such as formaters like sqlfluff then add the --dev tag:
+ ```bash
+ uv sync --group duckdb --dev
+ ```
+
+> Note: The --group and --dev arguments must be passed at the same time, passing them separately e.g `uv sync --group duckdb` `uv sync --dev` will just result in the last group specified being installed.
```

Comment:
is it possible to change this so we can install dependencies for multiple databases?  i would like to have both duckdb and postgres requirements installed.